### DISCUSSION TITLE
pypy2

### DISCUSSION THREAD

**Discussion by jayvdb** [ymyzk/tox-gh-actions#171] [Q&A] [answered]
**Describe the bug**
The docs here suggest using `pypy-2.7: pypy2-..`, however I believe it should be `pypy-2.7: pypy-..`.

Using 'pypy2' results in

```
| Running tox with tox-gh-actions
| $ tox
[CI/test]   ❓  ::add-matcher::/opt/hostedtoolcache/PyPy/2.7.18/x64/site-packages/tox_gh_actions/matcher.json
| No handlers could be found for logger "filelock"
| GLOB sdist-make: /Users/jayvdb/django/django-rest-framework-braces/setup.py
| pypy2-django11-drflatest create: /Users/jayvdb/django/django-rest-framework-braces/.tox/pypy2-django11-drflatest
| ::group::tox: pypy2-django11-drflatest
| ERROR: InterpreterNotFound:
| ___________________________________ summary ____________________________________
| ERROR:  pypy2-django11-drflatest: InterpreterNotFound:
| ::remove-matcher owner=tox-gh-actions-warning-error::
```

**Comment by ymyzk** [comment]
Hello, thanks for filing a ticket. tox-gh-actions itself is actually using `pypy-2: pypy2` and working correctly. https://github.com/ymyzk/tox-gh-actions/blob/2a9fdfcb4f591f229b5d576f5399087950fe923a/setup.cfg#L99

https://github.com/dealertrack/django-rest-framework-braces/pull/33/files is the PR you're working on? I can have a deeper look later when I have time to understand what's happening.

**Comment by jayvdb** [comment]
Yes, that is where I encountered it.
Thanks for looking.

**Comment by ymyzk** [comment, marked as answer]
@jayvdb, I think I could reproduce the issue you reported and find two different solutions.

https://github.com/ymyzk/django-rest-framework-braces/commit/6bba1db44d56b02990692c1b1ef48b451c16c60b 's the commit I used to reproduce the `InterpreterNotFound` issue. It seems this issue is caused by the `basepython` configuration under `[testenv]`

```
[testenv]
basepython =
    py27: python2.7
    py36: python3.6
    py37: python3.7
    py38: python3.8
    py39: python3.9
    py310: python3.10
    pypy: pypy
    pypy3: pypy3
```

IIUC, there are two solutions. Could you give it a try?

1. adding `pypy2: pypy2` to `basepython`: https://github.com/ymyzk/django-rest-framework-braces/commit/211c391db79c303fbdbf0f8fb356de123e32dfd1 / https://github.com/ymyzk/django-rest-framework-braces/actions/runs/2345994099
2. removeing the `basepython` completely: https://github.com/ymyzk/django-rest-framework-braces/commit/9e5e409d67d0db1b9cd24cc48b55e31765dc2536 / https://github.com/ymyzk/django-rest-framework-braces/actions/runs/2345996287
